wrangler-core/src/test/java/io/cdap/wrangler/TestingPipelineContext.java

```diff
@@ -142,4 +115,41 @@ public TransientStore getTransientStore() {
   public <T> Lookup<T> provide(String s, Map<String, String> map) {
     return null;
   }
+
+  @Override
+  public boolean schemaManagementEnabled() {
+    return schemaManagementEnabled;
+  }
+
+  protected class MockStageMetrics implements StageMetrics {
```

Comment:
nit: Any reason not to use `Mockito`? Much less boilerplate code

Reply:
Yeah that makes sense. I'd just moved the original code from the constructor to it's own class but I've used mockito now